inc/classes/dependencies/mobiledetect/mobiledetectlib/Mobile_Detect.php

```diff
@@ -22,1464 +22,1464 @@
  */
```

Comment:
Why is this file changed?

Reply:
I think this is pushed by mistake after running composer install, but I remember that I reverted it back, let me check